### PR TITLE
remove colorpicker lib and use html5 input color

### PR DESCRIPTION
**Pull request by pichalite**
reduces stylesheet size
uses html 5 input color element with built in support for color picker

### CODE REVIEW

**Comment by pichalite** [comment]
will follow up with theme PR's

File: public/language/en_GB/groups.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
 	"details.badge_preview": "Badge Preview",
 	"details.change_icon": "Change Icon",
 	"details.change_colour": "Change Colour",
+	"details.label_colour": "Badge BG Colour",
```
Comment (julianlam) [inline public/language/en_GB/groups.json]: Where is this string used? Corresponding theme PR?

**Comment by pichalite** [comment]
Holding this until Webkit supports `input type=color`

**Comment by pitaj** [comment]
#5964 for tracking

**Pull request closed without merging**